src/main/java/practice/StreamPractice.java

```diff
@@ -14,16 +24,27 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(number-> Arrays.stream(number.split(",")))
+                .toList()
+                .stream()
+                .mapToInt(a -> Integer.parseInt(a.trim()))
+                .filter(a -> a % 2 == 0)
```

Comment:
do not use 1-2 letter abbreviations even is streams, use understandable naming of variables